frontend/src/pages/projects/notebook/NotebookImagePackageDetails.tsx

```diff
@@ -25,7 +27,7 @@ const NotebookImagePackageDetails: React.FC<NotebookPackageDetailsProps> = ({
   return (
     <DescriptionList>
       <DescriptionListGroup>
-        <DescriptionListTerm>{title || 'Packages'}</DescriptionListTerm>
```

Comment:
you are just adding title back when using this component, why do you need includeTitle

I am guessing becuase the styling is different. Instead of having that `includeTitle` prop, open up the title prop to accept more than just string

`title?: string;` -> `title?: React.ReactNode;`

Then you can pass in your title component that has the correct heading size